test/integration/targets/vmware_tag/tasks/tag_manager_ops.yml

```diff
@@ -6,30 +6,30 @@
 - when: vcsim is not defined
   block:
     - name: Get VM Facts
-      vmware_vm_facts:
+      vmware_vm_info:
         hostname: '{{ vcenter_hostname }}'
         username: '{{ vcenter_username }}'
         password: '{{ vcenter_password }}'
         validate_certs: False
-      register: vm_facts
+      register: vm_info
 
-    - set_fact: vm_name="{{ vm_facts['virtual_machines'][0]['guest_name'] }}"
+    - set_fact: vm_name="{{ vm_info['virtual_machines'][0]['guest_name'] }}"
```

Comment:
This should be registered to a var for use in line 46, either in this or a follow-on PR.